README.md

```diff
@@ -607,6 +607,32 @@ dsn: clickhouse://dbuser:dbpass@hostname:9000/dbname
 
 See also: https://pkg.go.dev/github.com/ClickHouse/clickhouse-go
 
+**Databricks:**
+
+**Personal Access Token (PAT) Authentication:**
```

Comment:
To ensure consistency with other data sources, I would appreciate it if you could either discontinue the use of bold formatting or add explanatory comments within the code blocks.